Objects/classobject.c

```diff
@@ -71,7 +71,11 @@ method_vectorcall(PyObject *method, PyObject *const *args,
         }
         /* use borrowed references */
         newargs[0] = self;
-        memcpy(newargs + 1, args, totalargs * sizeof(PyObject *));
+        if (totalargs) { /* bpo-37138: if totalargs == 0, then args may be
```

Comment:
I don't think that the comment is required.

@gpshead: what do you think?

Reply:
I wrote the comment because the reason for that `if (totalargs)` is totally not obvious. It's an obscure point in the C standard that I was unaware of. I think that there are plenty of developers that do not know that `memcpy(dst, NULL, 0)` is undefined behaviour.

Reply:
I'm in favor of such comments. :)